index.html

```diff
@@ -131,12 +131,16 @@ <h2>What is a Hackathon?</h2>
                 <p>Students work in teams and mentors are present to offer help. Sponsors attend to help, recruit, and promote their products. Teams compete in different categories and prizes are awarded, but everyone learns and everyone wins!</p>
                 <p>Come spend 24 hours with us at BrickHack and dedicate time to learn, collaborate, build, and innovate.</p>
             </div>
-            <a href="https://mlh.io/" id="mlh" target="_blank" rel="noopener noreferrer">
-                <div>
+            <div id="hackathon-buttons">
+                <a href="https://mlh.io/" id="mlh" target="_blank" rel="noopener noreferrer">
                     <i class="fas fa-arrow-right"></i>
                     <p>Visit Major League Hacking (MLH)</p>
-                </div>
-            </a>
+                </a>
+                <a href="#" id="gallery" rel="noopener noreferrer">
```

Comment:
Clicking gallery currently scrolls to top of screen, but this code is the [allegedly](https://stackoverflow.com/questions/5637969/is-an-empty-href-valid) best way to do it.
```suggestion
                <a href="javascript:void(0);" id="gallery" rel="noopener noreferrer">
```